src/inputFile_impl.h

```diff
@@ -455,7 +512,14 @@ void InputFile<t_System>::create_lattice( Comm<t_System> *comm )
     T_X_FLOAT max_z = lattice_constant * lattice_nz;
     std::array<T_X_FLOAT, 3> global_low = { 0.0, 0.0, 0.0 };
     std::array<T_X_FLOAT, 3> global_high = { max_x, max_y, max_z };
-    system->create_domain( global_low, global_high );
+    if ( commandline.sparse )
```

Comment:
Since this commandline arg is already set up, can we input the multiplier here instead of assuming 2?

Reply:
Sure, I have missed this somehow :man_facepalming:

Reply:
I think that was the original behavior to get results, but this will be much more general

Reply:
Done. Example below with `1.5` multiplier (`mpiexec -n 8 ./build/bin/cbnMD -il input/in.lj --vacuum 1.5`).

![image](https://github.com/user-attachments/assets/216fa763-a1b1-4ec4-a6f5-6156b471d43c)